packages/salesforcedx-apex-debugger/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@salesforce/salesforcedx-apex-debugger",
+  "displayName": "Apex Debugger Adapter",
+  "description": "Implements VSCode debug protocol for Apex Debugger",
+  "version": "40.5.1",
+  "publisher": "salesforce",
+  "license": "BSD-3-Clause",
+  "engines": {
+    "vscode": "^1.15.0"
```

Comment:
The current plan is to keep people on the most recent version of VS Code. VS Code is still very new and has an aggressive release monthly cycle with each release including new APIs that we need to take advantage of. Case in point is the new APIs for debugging that we just introduced in 1.15.

See https://code.visualstudio.com/updates/v1_15#_debug-api-updates

The extensions indicate a minimum version that they need to be installed. If the user's version of VS Code is older than this, the user will not see the new versions of the extensions until she updates her VS Code.